VRCFaceTracking/Params/Expressions/Legacy/Eye/EyeTrackingParams.cs

```diff
@@ -58,18 +56,18 @@ public static class EyeTrackingParams
             #region EyeLidExpandedSqueeze
 
             new EParam(exp =>
-                exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight * .2f + exp.Eye.Left.Openness * .8f -
-                ((float)((1.0f - Math.Pow(exp.Eye.Left.Openness, .36)) * exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight)),
+                exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight * .2f + 
+                0.8f * exp.Eye.Left.Openness - exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight,
                 "LeftEyeLidExpandedSqueeze", 0.5f, true),
             new EParam(exp =>
-                exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight * .2f + exp.Eye.Right.Openness * .8f -
-                ((float)((1.0f - Math.Pow(exp.Eye.Right.Openness, .36)) * exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight)),
+                exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight * .2f +
+                0.8f * exp.Eye.Right.Openness - exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight,
                 "RightEyeLidExpandedSqueeze", 0.5f, true),
             new EParam(exp =>
-                (((exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight * .2f + exp.Eye.Left.Openness * .8f) + 
-                (exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight * .2f + exp.Eye.Left.Openness * .8f)) / 2.0f) -
-                (((float)((1.0f - Math.Pow(exp.Eye.Left.Openness, .36)) * exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight) + 
-                (float)((1.0f - Math.Pow(exp.Eye.Right.Openness, .36)) * exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight)) / 2.0f), 
+                (exp.Shapes[(int)UnifiedExpressions.EyeWideLeft].Weight * .2f +
+                0.8f *exp.Eye.Left.Openness - exp.Shapes[(int)UnifiedExpressions.EyeSquintLeft].Weight) + 
+                (exp.Shapes[(int)UnifiedExpressions.EyeWideRight].Weight * .2f +
+                0.8f * exp.Eye.Right.Openness - exp.Shapes[(int)UnifiedExpressions.EyeSquintRight].Weight) / 2.0f, 
```

Comment:
Not entirely sure why these are being reverted, you are going to run into the issue of Squint causing the eyelids to close pre-maturely. I am trying to have it so that squint doesn't affect the eyelids until the eyes are very nearly closed, this reversion will cause issues with 'proper' Unified implementations. It's just simply a glaring oversight with these ExpandedSqueeze parameters having Squint contending with 2 other parameters. It was built with the limitations of SRanipal in mind, and our v2 parameters will make sure that this combination isn't included. Also keep in mind the compensation won't apply to other eyelid params, even in the legacy params (EyeLidExpanded and the other uncombined ones).

Reply:
I am confused where the compensation should be handled at, I did it in ALVR tracking module for reducing the squint with the openness. Should this be handled in the module as it is sensitivity issue with multiple parameters? just removed all the power element with combined parameters.

Reply:
Only openness needs compensation, none of the other parameters will need it from ALVR module. This specific parameter type (ExpandedSqueeze) is the only one going to have the formula it has simply because it has squeeze and widen included in it, which are actively fighting against how the eye is being opened for this specific parameter. The ramp is there to prevent squeeze from affecting the shape until the eyes are closed (which will work around both the SRanipal tracking and AR52/FACS tracking). I think the ramp could be altered to be a lot more strong now that I have a clear idea on how the data should be sent from modules.

The only way I can explain how you should send the data from the modules is treat it like your targeting the values to be sent to the Unified (v2) parameters only (in which case you only need to use Lid Tightener and Eye Closed to control the Openness parameter like I did in the MeowFace formula), and after it works well for v2 then the SRanipal emulator will be developed to correct those parameters to work well for the old v1 (SRanipal) standard.

Hope that makes it clear, basically build the module to target v2 parameters and the SRanipal emulator will convert it to work for the old v1 parameters.